_rules/SC1-2-1-video-only-element-transcript.md

```diff
@@ -0,0 +1,133 @@
+
+---
+name: video only element transcript
```

Comment:
Shouldn't it be "video only element has transcript"?

Reply:
Done. Thanks.